weather_aplication/GeoDBJSON.swift

```diff
@@ -0,0 +1,61 @@
+import UIKit
+import SnapKit
+
+class GeoDBJSON: Decodable {
+    let data: [CityInfo]
+    let links: [Link]?
+    let metadata: Metadata
+
+    init(data: [CityInfo], links: [Link]?, metadata: Metadata) {
+        self.data = data
+        self.links = links
+        self.metadata = metadata
+    }
+}
+
+class CityInfo: Decodable {
+    let id: Int?
+    let wikiDataID, type, city, name: String?
+    let country, countryCode, region, regionCode: String?
+    let latitude, longitude: Double?
+    let population: Int?
+
+    enum CodingKeys: String, CodingKey {
+        case id
+        case wikiDataID = "wikiDataId"
+        case type, city, name, country, countryCode, region, regionCode, latitude, longitude, population
+    }
+
+    init(id: Int?, wikiDataID: String?, type: String?, city: String?, name: String?, country: String?, countryCode: String?, region: String?, regionCode: String?, latitude: Double?, longitude: Double?, population: Int?) {
+        self.id = id
+        self.wikiDataID = wikiDataID
+        self.type = type
+        self.city = city
+        self.name = name
+        self.country = country
+        self.countryCode = countryCode
+        self.region = region
+        self.regionCode = regionCode
+        self.latitude = latitude
+        self.longitude = longitude
+        self.population = population
+    }
+}
+
+class Link: Decodable {
+    let rel, href: String?
```

Comment:
jw

Reply:
dodatkowo - co to są za nazwy? jeżeli to skróty czy coś to nazwij je bardziej opisowo